### PR TITLE
Add if not option

### PR DESCRIPTION
**Pull request by kb22**
Now the question editor shows a toggle under show if option and the use if they want to show a question if a specific answer is selected or a specific answer is not selected.
This translates to the survey JSON as well.

### CODE REVIEW

**Comment by kb22** [comment]
The image shows the toggle option.

Screen Shot 2020-11-23 at 9 14 18 PM

**Review by evanlazaro — APPROVED** [review]
Looks good to me! But let's wait for feedback from @seandheffernan before merging

**Review by seandheffernan — APPROVED** [review]
Looks great! Thanks Karan!

**Pull request merged**
